client/extensions/woocommerce/state/sites/payment-methods/actions.js

```diff
@@ -107,3 +100,47 @@ export const savePaymentMethod = ( siteId, method, successAction = null, failure
 			}
 		} );
 };
+
+const savePaymentMethodEnabledSuccess = ( siteId, data ) => {
+	const paymentMethod = addPaymentMethodDetails( data );
+	return {
+		type: WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS,
+		siteId,
+		data: paymentMethod,
+	};
+};
+
+export const savePaymentMethodEnabled = (
+	siteId,
+	method,
+	enabled,
+	successAction = null,
+	failureAction = null
+) => ( dispatch, getState ) => {
+	const state = getState();
+	if ( ! siteId ) {
+		siteId = getSelectedSiteId( state );
```

Comment:
You might like https://github.com/Automattic/wp-calypso/blob/e1f9cddd82b2ca92e1e3f20f7cbec36812ea088b/client/extensions/woocommerce/state/sites/selectors.js#L16 instead